test/README.md

```diff
@@ -215,6 +215,13 @@ Change directory to the test directory:
 cd test
 ```
 
+To setup up local kubernetes cluster for drenv, run once before starting any environment:
+
+```
+drenv setup envs/regional-dr.yaml
+```
+Note: Replace "envs/regional-dr.yaml" with your personalised setup file to configure custom k8s cluster environment.
```

Comment:
This may be confusing since you don't need to replace anything in the common case. Maybe something like:

    Note: to set up another environment file, replace regional-dr.yaml with the actual file.

But I'm not sure this is really needed, it may confuse instead of help.

Reply:
Removed the note as recommended.